digital-deck-server/__tests__/playerTests.js

```diff
@@ -0,0 +1,86 @@
+const request = require('supertest');
+const app = require('../index');
+const {addSession} = require('../services/utilities');
+
+beforeAll(() => {
+    var session = {
+        "numDecks" : 1,
+        "numPlayers" : 1, 
+        "sessionId" : 0,
+        "players" : {
+            "cards" : [],
+            "playerId" : 0
+        },
+        "table" : [],
+        "deck" : [1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52]
+    };
+    addSession(session);
+});
+
+/**
+ * Testing draw card endpoint
+ */
+
```

Comment:
Since the project structure is quite different, I couldn't verify if these tests work. @dhillonJas After I merge, it would be great if you can rebase your branch and verify these.